app/routes/signups.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router(); // eslint-disable-line new-cap
```

Comment:
I found new-cap rule not very effective. Might make sense to just turn it off for properties for good:
https://github.com/DoSomething/quicksilver-api/blob/b69f8c90fa1901cfc1fb250262236071e56a43b9/test/.eslintrc#L8-L10